packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -45,6 +47,59 @@ interface EnqueueTaskParams {
   name: string;
 }
 
+interface LogFn {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I've disabled linting for the `any[]` in these lines because this is the signature for the underlying Pino logger and this seemed like the most elegant way to wrap it. If we are more specific about types then we'll have to add a bunch of code to transform the inputs before passing to Pino